README.md

```diff
@@ -621,7 +619,10 @@ echo "Operation status: ".$status->getStatus()."<br />";
 ```
 
 ##Media Services
-
+> **Note**
+> 
+>The current Media Services API only works with PHP 5.5 and above.
```

Comment:
IMO, if any part of this library requires PHP 5.5 or higher (no matter if it is just one component), the requirement should be updated at [`composer.json`](https://github.com/Azure/azure-sdk-for-php/blob/c25f75f5d4c1244c3cdd65dfe2f53f2d5e435d61/composer.json#L13).